django_app/redbox_app/templates/accessibility-statement.html

```diff
@@ -1,46 +1,39 @@
-{% import "macros.html" as macros %}
-{% extends "base_generic_gov.html" %}
+{% extends "base.html" %}
 
-<!--TODO: Update system url and other variables-->
-
-{% block title %}
-  <title>Accessibility statement - Redbox Copilot - GOV.UK</title>
-{% endblock %}
+{% set pageTitle = "Accessibility statement" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot Platform</h1>
-  <div class="govuk-body">
-    <p>This accessibility statement applies to Redbox Copilot - <a href="<SYSTEM_URL>" class="govuk-link"><SYSTEM_URL></a>.</p>
-    <p>This website is run by the Cabinet Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
+  <h1 class="govuk-heading-l">Accessibility statement for Redbox Copilot</h1>
+
+    <p class="govuk-body">This accessibility statement applies to Redbox Copilot.</p>
+    <p class="govuk-body">This website is run by the Cabinet Office. We want as many people as possible to be able to use this website. For example, that means you should be able to:</p>
     <ul class="govuk-list govuk-list--bullet">
       <li>change colours, contrast levels and fonts</li>
       <li>zoom in up to 300% without the text spilling off the screen</li>
       <li>navigate most of the website using just a keyboard</li>
       <li>navigate most of the website using speech recognition software</li>
       <li>listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
-  </ul>
-  <p>We've also made the website text as simple as possible to understand.</p>
-  <p><a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    </ul>
+    <p class="govuk-body">We've also made the website text as simple as possible to understand.</p>
+    <p class="govuk-body"><a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+  
     <h2 class="govuk-heading-m">Feedback and contact information</h2>
-    If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
-      <ul class="govuk-list govuk-list--bullet">
-        <li>email <a href="mailto:redbox-copilot-support@cabinetoffice.gov.uk" class="govuk-link">redbox-copilot-support@cabinetoffice.gov.uk</a></li>
-      </ul>
-    <p>We'll consider your request and get back to you in 14 days.</p>
-
+    <p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, email <a href="mailto:redbox-copilot@cabinetoffice.gov.uk" class="govuk-link">redbox-copilot@cabinetoffice.gov.uk</a></p>
```

Comment:
You should be able to use the `{{CONTACT_EMAIL}}` env var if we have it in this project to make it easier to update if it changes

Reply:
Yeah, that's a good point. I'll take a look!

Reply:
I've added that in now, and added the correct email address to the example .env files